backend/core/scripts/ami-chart-import-files/Oakland-2023.txt

```diff
@@ -0,0 +1,12 @@
+20	20700	23666	26633	29566	31933	34300	36666	39033
```

Comment:
Do we need to bring this file into the repo? Once the Oakland-2023.ts file is generated this txt file is no longer needed I'm assuming?

Reply:
yeah I think in the future we can opt to not commit this file, I think that since this is the first one it'll be a decent example of what we want the file format to be for the import script